src/components/Books/Books.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import NewBook from '../NewBook/NewBook';
+import {
+  BookWrapper, BookDivLeft, BookDivRight,
+  BookDivCenter, ButtonsContainer, Gauage, PercentageSpan, StatusSpan,
+  Category, Title, Author, BooksButton, CurrentChapter,
+  Chapter, UpdateButton,
+} from './BooksElements';
```

Comment:
- [ ] Nice implementation you have here 🔥. Keep rocking.

Reply:
Thank you for your kind comment 😃  ❤️ 🤝